app/views/static_pages/help.html.erb

```diff
@@ -0,0 +1,10 @@
+<%= provide :title, "Help" %>
+
+<h1>Help</h1>
+  <p>
+    Get help on the Ruby on Rails Tutorial at the 
+     <%= link_to "Rails Tutorial help page", Settings.links.rails_tutorial.help_page %>
+      
+     To get help on this sample app, see the
+    <%= link_to "<em>Ruby on Rails Tutorial</em>book", Settings.links.rails_tutorial.book_page %>
+  </p>
```

Comment:
remove 2 spaces

Reply:
done but can you check again if I do it correctly or not